roles/edpm_ovn/tasks/configure.yml

```diff
@@ -41,12 +41,24 @@
   block:
     - name: Set enable-chassis-as-gw
       ansible.builtin.set_fact:
-        cms_options:
+        gw_cms_options:
```

Comment:
(not your fault, just pointing out for future) we could probably avoid setting these facts only to later combine them into external_ids map. We could probably just combine in place.

Reply:
Indeed. Won't tackle it in this PR but it makes sense